.github/workflows/continuous-integration.yml

```diff
@@ -13,12 +13,7 @@ jobs:
     steps:
       # Checks-out the repository under $GITHUB_WORKSPACE, so the job can access it
       - uses: actions/checkout@v4
-
-      - name: Setup Bazel
-        uses: abhinavsingh/setup-bazel@v3
```

Comment:
this isn't needed since GitHub Actions runner images have bazelisk installed by default.

When/If we want a fancier caching setup, we can use a different action that's being upstreamed to bazel-contrib right now.